spec/features/delete_answer_spec.rb

```diff
@@ -25,12 +25,12 @@
     sign_in(answer.user)
     visit question_path(other_question)
 
-    expect(page).to_not have_selector(:link_or_button, 'Delete question')
+    expect(page).to_not have_selector(:link_or_button, 'Delete answer')
   end
 
-  scenario "An unauthenticated user can not delete question" do
+  scenario "An unauthenticated user can not delete answer" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.